docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
-Manage scratch orgs
+Manage Scratch Orgs
 ===================
 
-What is an org?
----------------
+Scratch orgs are temporary Salesforce orgs which can be quickly set up "from scratch," and which last for no more than 30 days. There are several reasons why scratch orgs are encouraged for development and testing over sandboxes or Developer Edition orgs.
 
-Set up the Salesforce CLI
--------------------------
+* Scratch orgs provide a repeatable starting point without the challenge of managing persistent orgs' state over time.
+* Scratch orgs are scalable and ensure that individual, customized environments are available to everyone in the development lifecycle.
+* Scratch orgs facilitate a fully source-driven development process built around best practices.
+
+CumulusCI offers tools for working with all types of Salesforce orgs, but provides the most value when working with scratch orgs. CumulusCI automation helps realize the promise of scratch orgs as low cost, repeatable, source-driven environments for every phase of the product lifecycle.
+
+This section focuses on managing scratch orgs in a CumulusCI project. To learn about managing persistent orgs, such as sandboxes, production orgs, and packaging orgs, read `Connect persistent orgs<TODO>`_.
+
+
+
+What Is an Org in CumulusCI?
+----------------------------
+
+An org in CumulusCI's keychain starts out as a named configuration, tailored for a specific purpose within the lifecycle of the project (such as development, feature, beta, and so on). CumulusCI creates and uses scratch orgs that are portable and repeatable. In fact, a scratch org is only generated the first time you use the scratch org. When it's expired or been deleted, a new one can easily be created again with the same configuration file.
```

Comment:
Several times throughout this section "the configuration" is mentioned. I spoke to @Br4nd0R about it, and we settled on "configuration file". I'd like to get a consensus on this before I go changing every single instance of it.

Reply:
I agree that the terminology is confusing, but I don't think that "configuration" == "configuration file". A scratch org configuration includes both markup in `cumulusci.yml` and in the `orgs/` directory, and goes by a specific name (e.g., `dev`).

Reply:
This paragraph is incorrect in using "configuration" once and "configuration file" another time. It should be "configuration" both times.

Reply:
@davidmreed Yeah, that's on me. I spoke with @Br4nd0R about clarifying this particular sentence, knowing that there would need to be further discussion on "configuration". I'll make sure it reflects the other instances when we sort out terminology.

Reply:
Gonna close this conversation, but we will have to follow up on whether we say "configuration" or "configuration file" throughout this entire section.